React/tic-tac/src/redux/auth/actions.js

```diff
@@ -0,0 +1,14 @@
+function LogInTrue(payload) {
+  return {
+    type: 'LogIn',
```

Comment:
Define types in a different object so it'd be easier for you to replace them when a change needs to be done. Something like this (this is just an example, you may have to adapt it to your own code):
```
const actions = {
  LOG_IN: 'LOG_IN'
}
```
Then reference it like: 
``` 
  dispatch({ type: actions.LOG_IN, payload: SOMETHING });
```